### PR TITLE
[DNM] Add tempest job for ironic operator - extra debug for troubleshooting.

### PR DESCRIPTION
**Pull request by hjensas**

### CODE REVIEW

**Comment by hjensas** [comment]
/pj-rehearse pull-ci-openstack-k8s-operators-ironic-operator-master-ironic-operator-build-deploy-tempest

**Comment by hjensas** [comment]
/pj-rehearse pull-ci-openstack-k8s-operators-ironic-operator-master-ironic-operator-build-deploy-tempest

**Comment by son-vyas** [comment]
/pj-rehearse pull-ci-openstack-k8s-operators-ironic-operator-main-ironic-operator-build-deploy-tempest

File: ci-operator/step-registry/openstack-k8s-operators/tempest/openstack-k8s-operators-tempest-commands.sh [hunk 1]
```diff
@@ -63,6 +67,92 @@ if [[ "${SERVICE_NAME}" == "MANILA" ]]; then
...
+EOF
+
+git clone https://opendev.org/openstack/tempest.git ~/git/tempest
+pushd ~/git/tempest/
+git apply ~/git/0001-DNM-Debug-auth-catalog-endpoint-filtering.patch
+popd
+git clone https://opendev.org/openstack/ironic-tempest-plugin.git ~/git/ironic-tempest-plugin
+mkdir ~/tempest-venv
+python3 -m venv ~/tempest-venv --system-site-packages
+source ~/tempest-venv/bin/activate
```
[the remark this answers is not in the document]
Comment (hjensas) [reply ci-operator/step-registry/openstack-k8s-operators/tempest/openstack-k8s-operators-tempest-commands.sh]: Let's see how it goes with this new revision.

**Comment by hjensas** [comment]
/pj-rehearse pull-ci-openstack-k8s-operators-ironic-operator-main-ironic-operator-build-deploy-tempest

**Comment by hjensas** [comment]
/pj-rehearse pull-ci-openstack-k8s-operators-ironic-operator-main-ironic-operator-build-deploy-tempest

File: ci-operator/step-registry/openstack-k8s-operators/tempest/openstack-k8s-operators-tempest-commands.sh (hunk 1 above)
[the remark this answers is not in the document]
Comment (son-vyas) [reply ci-operator/step-registry/openstack-k8s-operators/tempest/openstack-k8s-operators-tempest-commands.sh:151]: @arxcruz can you please look into this

**Comment by hjensas** [comment]
/pj-rehearse pull-ci-openstack-k8s-operators-ironic-operator-main-ironic-operator-build-deploy-tempest

**Comment by hjensas** [comment]
/pj-rehearse pull-ci-openstack-k8s-operators-ironic-operator-main-ironic-operator-build-deploy-tempest

**Pull request closed without merging**